samvera-labs/bulkrax
# 195 -fix browse everything button

# Summary
- [ ] Clicking on the "Add Çloud Files" button should allow the user to select files via the browse everything modal.

## BEFORE
Clicking the "add cloud files" button would do nothing.

## AFTER
# Screenshots / Video

https://user-images.githubusercontent.com/10081604/182660454-56b8fe25-90a5-4a7d-b60b-1323d086555e.mp4



# Testing Instructions
1. Create a new bulkrax importer
2. select csv parser.
3. click "add cloud files" button and select jpgs. After adding them you should see them listed in the bottom and the button should turn green. 

# Notes

- [ ] You may need to refresh cache
- [ ] There are some known related bugs that have been filed against bulkrax. For instance, if you refresh the page after selecting a parser, the button won't work again until you reselect the parser.

For now, the user will have to re select the parser to make the button work again.

REF: https://gitlab.com/notch8/utk-hyku/-/merge_requests/80
